src/schema/imported/tabs.json

```diff
@@ -361,7 +362,7 @@
             },
             "highlighted": {
               "type": "boolean",
-              "description": "Whether the tabs are highlighted."
+              "description": "Whether the tabs are highlighted.  Works as an alias of active."
```

Comment:
doh! double whitespaces! I know it's a import but things like that jump directly into my eyes